analyzers/src/SonarAnalyzer.Common/Extensions/LocationExtensions.cs

```diff
@@ -44,4 +44,7 @@ public static Location EnsureMappedLocation(this Location location)
 
     public static int EndLine(this Location location) =>
         location.GetLineSpan().EndLinePosition.Line;
+
+    public static SecondaryLocation ToSecondary(this Location location, string message = null) =>
```

Comment:
All extension methods should all be named the same: either `ToSecondaryLocation` or `ToSecondary`
```suggestion
    public static SecondaryLocation ToSecondaryLocation(this Location location, string message = null) =>
```

Reply:
I was considering this from a reader point of view. Like `location.ToSecondary()` is good by itself, and `node.ToSecondaryLocation()` is self-explanatory. As `node.ToSecondary()` seem insufficient to me - it's not clear what you should expect as a secondary (node?).

So the current state is one option, and `location.ToSecondaryLocation()` is the other one. I find the "Location" superfluous in this case.

I can extend the 2nd name. Do you still want to do it, or is the reasoning enough to keep it?

Reply:
I was proposing elsewhere adding `ToSecondaryLocation` extension methods for
* `SyntaxToken` and
* `IEnumerable<SyntaxToken|SyntaxNode|Location>`
* maybe also `SyntaxReference` and `DeclaringSyntaxReferences`

If you follow that advice, I would name them all the same. If not, we can keep the two names, even though I still prefer them the same.